web_src/less/_base.less

```diff
@@ -1,3 +1,5 @@
+@import "./variables.less";
```

Comment:
Move this import to `index.less`.

Reply:
Probably only need to import it once, unless Less variables are scoped per file, but think the act globally.